exercises/concept/temperature/test/Tests.hs

```diff
@@ -0,0 +1,42 @@
+import           Control.Monad (forM_)
+import           Temperature   (tempToC, tempToF)
+import           Test.Hspec    (describe, hspec, it, shouldBe)
+import           Text.Printf   (printf)
+
+inputTtoF :: [Float]
+inputTtoF  =  [-100.00, -18.00, 0.00, 50.10]
+
+inputTtoC :: [Integer]
+inputTtoC = [-148, 0, 32, 123]
+
+expectTtoC :: [Float]
+expectTtoC = [-100.00, -17.78, 0.00, 50.56]
+
+expectTtoF :: [Integer]
+expectTtoF = inputTtoC
+
+answersTtoC :: [Float]
+answersTtoC = map tempToC inputTtoC
+
+answersTtoF :: [Integer]
+answersTtoF = map tempToF inputTtoF
+
+testTtoF :: [(Float, Integer, Integer)]
+testTtoF = zip3 inputTtoF answersTtoF expectTtoF
+
+testTtoC :: [(Integer, Float, Float)]
+testTtoC = zip3 inputTtoC answersTtoC expectTtoC
+
+main :: IO ()
+main =  hspec $ do
+        describe "tempToF:" $ do
+          forM_ testTtoF $ \(inputTemp, ans, expected) ->
+            it (printf "%.2f C equals %i F" inputTemp ans) $
+            ans `shouldBe` expected
+        describe "tempToC:" $ do
+          forM_ testTtoC $ \(inputTemp, ans, expected) ->
+            it (printf "%i F equals %.2f C" inputTemp ans) $
+             roundTo 2 ans `shouldBe` expected
```

Comment:
I don't know if other exercises use this parameterized form of testing. Maybe @petertseng knows?

Reply:
I took the idea from another exercise. In the Elm track I think.

Reply:
I've checked existing exercises, and they don't seem to use the parameterized tests option. See https://github.com/exercism/haskell/blob/main/exercises/practice/anagram/test/Tests.hs and https://github.com/exercism/haskell/blob/main/exercises/practice/bob/test/Tests.hs. 
Could you convert to regular, non-parameterized tests?

Reply:
You've marked this as resolved, but I don't see any change. Is that correct?